protocol/messenger_contacts.go

```diff
@@ -30,7 +30,7 @@ func (m *Messenger) prepareMutualStateUpdateMessage(contactID string, updateType
 		case MutualStateUpdateTypeSent:
 			text = "You sent a contact request to @" + to
 		case MutualStateUpdateTypeAdded:
-			text = "You added  @" + to + " as a contact"
+			text = "You accepted @" + to + "'s contact request"
```

Comment:
@MishkaRogachev since this text needs to be translated, isn't better to set it a bit differently? Unique constant? Also since it's always a case only for the 1:1 chat there is no need to add "to"/"from" anywhere in it (the same below).

Reply:
fixed, btw there was a discussion on this topic here: https://github.com/status-im/status-go/pull/3519#discussion_r1206799703